app/src/main/java/com/example/yard/activities/PostActivity.java

```diff
@@ -70,12 +72,16 @@ protected void onCreate(Bundle savedInstanceState) {
     Button btnRemoveLocation = findViewById(R.id.btnRemoveLocation);
     fusedLocationProviderClient = LocationServices.getFusedLocationProviderClient(this);
     btnGetLocation.setOnClickListener(view -> getLastLocation());
-    btnTakePicture.setOnClickListener(view -> launchcamera());
+    btnTakePicture.setOnClickListener(view -> launchCamera());
     ivImage.setOnTouchListener(this);
     gestureDetector = new GestureDetector(this, this);
     Toast.makeText(
             this, " SWIPE left or right to DELETE a photo after taking it!", Toast.LENGTH_LONG)
         .show();
+    btnUpload.setOnClickListener(
+        view -> {
```

Comment:
Let's not include changes that is not related to account deletion. You might to rebase this branch on top of the reformating branch.